src/core_codemods/docs/pixee_python_flask-json-response-type.md

```diff
@@ -0,0 +1,16 @@
+The default `mimetype` for `make_response` in Flask is `'text/html'`. This is true even when the response contains JSON data.
+If the JSON contains (unsanitized) user-supplied input, a malicious user may supply HTML code which leaves the application vulnerable to cross-site scripting (XSS). 
+This fix explicitly sets the response type to `application/json` when the response body is JSON data to avoid this vulnerability. Our changes look something like this:
+
+```diff
+from flask import make_response, Flask
+import json
+
+app = Flask(__name__)
+
+@app.route("/test")
+def foo(request):
+    json_response = json.dumps({ "user_input": request.GET.get("input") })
+-   return make_response(json_response)
++   return (make_response(json_response), {'Content-Type':'application/json'}
```

Comment:
See other comments regarding the overall conditions for this change, but I think it would be more idiomatic and more aligned with the purpose of `make_response` to call it like this:
```python
return make_response(json_response, {'Content-Type':'application/json'})
```

See the documentation and specifically the `tuple` parameter case: https://flask.palletsprojects.com/en/3.0.x/api/#flask.Flask.make_response

Reply:
The reason for the tuple response is because it is more general. It will also work for the direct `json.dumps` response. I'll adjust the transformation for the `make_response` case.